torch/nn/modules/module.py

```diff
@@ -20,6 +20,20 @@ def _addindent(s_, numSpaces):
     return s
 
 
+def _flatten(*args):
```

Comment:
nit: I wonder if @ezyang was talking about [this `_flatten`](https://github.com/pytorch/pytorch/blob/master/torch/jit/__init__.py#L540-L543)